src/i-configuration.ts

```diff
@@ -63,6 +63,8 @@ export class StoreBackedConfiguration implements IConfiguration {
       );
     }
     await Promise.all(promises);
+
+    // TODO: notify of config change if `didUpdateFlags` is true
```

Comment:
This PR or a future one?

Reply:
Future one (this TODO has been moved around a couple of times now).

All of this config refactoring is leading to this line being called _every time_ the configuration changes. Then we can implement the `onConfigurationChange` and `waitForInitialization` methods